README.md

```diff
@@ -60,7 +64,7 @@ Help (from cmd or powershell)
 
 ```
 Usage:
-  bt2qbt_v1.999_amd64.exe [OPTIONS]
+  bt2qbt_v1.99_amd64.exe [OPTIONS]
```

Comment:
Why? :-)

Reply:
All your releases are so far x.yy digit long.
https://github.com/rumanzo/bt2qbt/releases

Reply:
I was thinking about this for at least 15 minutes long, if I should change or not.
My other idea was:
`bt2qbt_v.........._amd64.exe`

Reply:
It's doesn't matter actually, just it should be not equal actual releases for avoid copy paste mistakes.
Just asking)